guides/release/models/customizing-adapters.md

```diff
@@ -230,17 +235,18 @@ property dependent on the `session` service.
 import DS from 'ember-data';
 import { computed } from '@ember/object';
 import { inject as service } from '@ember/service';
+const { JSONAPIAdapter } = DS;
 
-
-export default DS.JSONAPIAdapter.extend({
-  session: service('session'),
-  headers: computed('session.authToken', function() {
+export default class ApplicationAdapter extends JSONAPIAdapter {
+  @service session;
+  @computed('session.authToken')
```

Comment:
I think we can remove `@computed` here and just make this a tracked property

Reply:
Rather, `session.authToken` should be considered a tracked property, and this can just be a normal getter